hw_5/task_2.py

```diff
@@ -0,0 +1,10 @@
+# Напишите однострочный генератор словаря, который принимает на вход три списка одинаковой длины:
+# имена str, ставка int, премия str с указанием процентов вида «10.25%».
+# В результате получаем словарь с именем в качестве ключа и суммой премии в качестве значения.
+# Сумма рассчитывается как ставка умноженная на процент премии
+
+names = ['Сергей Владимирович', 'Илон Маск', 'Чанпэн Чжао']
+rates = [100000, 2005000, 15052255]
+bonuses = ['10.25%', '5%', '12.5%']
+
+print({name: rate * float(bonus.strip('%')) / 100 for name, rate, bonus in zip(names, rates, bonuses)})
```

Comment:
Абсолютно верно, но избавиться от символа процента можно проще
```suggestion
print({name: rate * float(bonus[:-1]) / 100 for name, rate, bonus in zip(names, rates, bonuses)})
```